src/npc.h

```diff
@@ -552,6 +552,10 @@ class npc : public player
  npc &operator=(const npc &) = default;
  npc &operator=(npc &&) = default;
  ~npc() override;
+
+    /** Construct an NPC with standard stats */
+    static npc standard();
```

Comment:
I don't particularly like it here. This function is used only for debugging purposes (statistics, testing) and the name looks like a getter to me, but it's a constructor.